app/vmalert/storage/storage.go

```diff
@@ -0,0 +1,11 @@
+package storage
```

Comment:
I'd suggest to rename `storage` int `datasource` or smthng alternative. The `datasource` should be an interface, just like `provider` package. This would be helpful for adding new datasources in future.